components/layouts/content-list/content-list--years.hbs

```diff
@@ -0,0 +1,79 @@
+<div class="section--bg-prim-blue">
+  <div class="content-list section__inner">
+    <h1 class="content-list__headline">Where we're at today.</h1>
+
+    <div class="content-years__preface themed-text">
```

Comment:
I'd really like to see us do this without any custom BEM here. The important element is that stat panel below, and it only occurs in this one instance, so the solution is going to be using basic content, with the markup for the stats included inline for this one instance. All the before and after text will just be plain old content like any other generic container. The h1 here will be a standard section-heading like anywhere else.